content/departments/technical-success/ce/process/license_keys.md

```diff
@@ -28,12 +28,13 @@ Sourcegraph requires site administrators to apply a license key to access to var
 License keys can only be generated by site administrators on sourcegraph.com. To create or maintain a license you must request temporary access via Entitle (in Okta). Entitle grants temporary access to site admin.
 
 1. Go to Entitle via Okta
-2. Search for `site admin` and select dotcom site admin access. Access is logged but automatically provisioned.
-3. Log out of your sourcegraph.com/search account
-4. Go [here](https://sourcegraph.com/sign-in?sourcegraph-operator) and select `other login methods`.
-5. Click `Continue with Sourcegraph Operators`. You'll be signed in and taken to sourcegraph.com/search.
-6. On the top right click on your profile icon and the `site admin` menu will now be available for you to access.
-7. Go to `Enterprise licenses`.
+2. Search for `site admin` and select dotcom site-admin. Access is logged but automatically provisioned.
```

Comment:
Can you add a deep link to this bundle, or are we avoiding that for public-facing docs? If that's the case, maybe a Go link?

Reply:
I will update this if we decide to put in the deep link but for now going to merge like this. once our discussion in the team channel had a conclusion, ill circle back to it. Thank you for the suggestion!